kontrakter/dom/begjaeringDom/1.0/begjaeringDom.schema.json

```diff
@@ -0,0 +1,671 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://politiet.no/dom/1.0/begjaeringDom",
+  "description": "Begjæring om dom fra politiet til tingretten, ønsker dom avgjørelse tilbake.",
+
+  "type": "object",
+  "properties": {
+    "forsendelse": {
+      "$ref": "#/definitions/forsendelse",
+      "description": "avsender og mottaker"
+    },
+    "begjaeringDom": {
+      "$ref": "#/definitions/begjaeringDom"
+    },
+    "hovedStraffesak": {
+      "$ref": "#/definitions/straffesak",
+      "description": "Hoved straffesaken saksbehandler saken."
+    },
+    "dokumenter": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/dokument"
+      }
+    }
+  },
+  "required": ["forsendelse", "hovedStraffesak", "dokumenter"],
+  "additionalProperties": false,
+
+  "definitions": {
+    "begjaeringDom": {
+      "type": "object",
+      "description": "Begjæringen med basisdata. Detaljer i egne meldinger for siktelse/tiltale og påstand om straff",
+      "properties": {
+        "kravId": {
+          "$ref": "#/definitions/GUID",
+          "description": "Krav om avgjørelse til domstolen, tingretten ønsker avgjørelse (dom tilbake)"
+        },
+        "begjaeringstype": {
+          "type": "string",
+          "description": "Hvilken sak skal dette være, se readme for forklaring. Kun TILSTAAELSE i første omgang.",
+          "enum": [
+            "TILSTAAELSE",
+            "TILSTAAELSE_FORERETT",
+            "TILSTAAELSE_VARETEKT",
+            "MEDDOMSSAK"
+          ]
+        },
+        "ansvarligPaatalejurist": {
+          "$ref": "#/definitions/paataleJurist",
+          "description": "Ansvarlig påtalejurist på straffesaken, kanskje ikke den som har skrevet begjæringen"
+        },
+        "siktedePersoner": {
```

Comment:
Ønsker vi å ha med siktede personer/foretak både i begjæring og siktelseTiltale? Hva om det er diff - hvilken er det som gjelder?

Reply:
Har med siktede for at meldingen skal stå på egne bein, dvs. før siktelsen kommer. Det skal være samme data på de siktede.

Reply:
Det vil være en felles nøkkel som finnes i siktelsen og begjæringen. 
Vi vil sende data nesten samtidig, men siktelsen kan være noen dager/uker gammel. Planen er å sende med oppdaterte persondata også på siktelsen, men vi har ikke diskutert hva vi gjør hvis det er store forskjeller.

Reply:
Begjæringen kommer med siste oppdaterte data på siktede, vitner, fornærmede, verger.....